pr_etiquette.md

```diff
@@ -0,0 +1,62 @@
+# Etiquette for pull requests
+
+## As an author
+
+Submitting pull requests in EDC should be done while adhering to a couple of simple rules.
+
+- Familiarize yourself with [coding style](styleguide.md), [architectural patterns](docs/architecture-principles.md) and
+  other contribution guidelines.
+- No surprise PRs please. Before you submit a PR, open a discussion or an issue outlining your planned work and give
+  people time to comment. It may even be advisable to contact committers using the `@mention` feature. Unsolicited PRs
+  may get ignored or rejected.
+- Create focused PRs: your work should be focused on one particular feature or bug. Do not create broad-scoped PRs that
+  solve multiple issues as reviewers may reject those PR bombs outright.
+- Provide a clear description and motivation in the PR description in GitHub. This makes the reviewer's life much
+  easier. It is also helpful to outline the broad changes that were made, e.g. "Changes the schema of XYZ-Entity:
+  the `age` field changed from `long` to `String`".
+- If you introduce new 3rd party dependencies, be sure to note them in the PR description and explain why they are
+  necessary.
+- Stick to the established code style, please refer to the [styleguide document](styleguide.md).
```

Comment:
Sticking to the code style is already addressed in the first rule. Do we need the second reference?